Formula/ansible@2.9.rb

```diff
@@ -26,6 +19,9 @@ class AnsibleAT29 < Formula
 
   keg_only :versioned_formula
 
+  # EOL: https://docs.ansible.com/ansible/latest/reference_appendices/release_and_maintenance.html
+  deprecate! date: "2022-05-31", because: :versioned_formula
```

Comment:
2.9 was listed as planned for EOL May 2022. Current docs list it as unmaintained/EOL now (https://docs.ansible.com/ansible/latest/reference_appendices/release_and_maintenance.html#ansible-core-changelogs).

Reply:
Let's put it down as this then:
```suggestion
  # EOL: https://docs.ansible.com/ansible/latest/reference_appendices/release_and_maintenance.html
  disable! date: "2023-05-31", because: :unmaintained
```

Reply:
I went with `2023-05-11` after finding upstream EOL commit https://github.com/ansible/ansible/commit/6230244537b57fddf1bf822c74ffd0061eb240cc